curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/653fb19b515fde28243f727a.md

```diff
@@ -7,29 +7,22 @@ dashedName: step-11
 
 # --description--
 
-Now you need a way to display the songs in the UI. To do this, you'll create a `renderSongs` function using the arrow function syntax.
+Now you need a way to display the songs in the UI(User Interface). To do this, you'll create a `renderSongs` function using the arrow function syntax.
```

Comment:
```suggestion
Now you need a way to display the songs in the UI (User Interface). To do this, you'll create a `renderSongs` function using the arrow function syntax.
```
Nitpick

Reply:
@gikf fixed the spacing within the description